packages/resource-deployment/templates/on-demand-scan-req-schedule.template.json

```diff
@@ -4,7 +4,7 @@
         "recurrenceInterval": "PT2M"
     },
     "jobSpecification": {
-        "priority": 0,
+        "priority": 1000,
```

Comment:
P2: i would suggest not to use the maximum value. instead use a smaller value that is greater than all other jobs we currently have, since if we want to have another job with higher priority, this will make us to reset this priority first.

Reply:
I don't think we will have another job that has higher priority than scan request sender, and if we do that means we should assign it in another pool or at least change the pool name